README.md

```diff
@@ -1,267 +1,189 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+> This application is built with PostgreSQL as a Database.
+<a href="https://www.postgresql.org/" target="_blank" rel="noreferrer"> <img
+      src="https://user-images.githubusercontent.com/57408419/178756727-8bfad119-18c2-49eb-98be-8b2d4bd71dd6.jpeg"
+      alt="react" width="120" height="70" /> </a>
+
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby - A Programmer's Best Friend</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[User identifcations]**
+- **[Can view all categories ]**
+- **[User can see how much spent in each each categorie]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://moneymanager-342g.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Video Presentation
+
+[video link](https://drive.google.com/file/d/16Qktky0anaig0xfyzDlhOH9ppj0Bn5Og/view?usp=sharing)
+
+<!-- I have trying for days now to record on loom when i do the recording it fails to upload it to loom so i used zoom -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+> In order to run this project you need:
+> - Download and Install [Ruby](https://www.ruby-lang.org/en/downloads/) for your OS.
+> - Download and Install [PostgreSQL](https://www.postgresql.org/download/)
+> - VS Code
+> - Ruby on Rails
+```sh
+  gem install rails
+```
+
+### Setup
+
+> Clone this repository to your desired folder:
+```sh
+  git clone https://github.com/hydaline-code/expense_tracker.git
+```
+```sh
+  cd expense_tracker
+```
+
+### Install
+
+> Install this project with:
+```sh
+  gem install
+```
+
+```sh
+  bundle install
+```
+
+### Usage
+
+> To run the project, execute the following command:
+```sh
+  rails server
+```
+
+### Run tests
+
+> To run linters check, run the following command:
+
+```sh
+  gem install rubocop
+```
+
+```sh
+  rubocop
+```
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Hydaline Charlene**
+
+> - GitHub: [hydaline](https://github.com/hydaline-code)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Interactive screen ]**
+- [ ] **[Edit and delete features ]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).

```

Comment:
- [ ] Please make sure that you don't have any broken links in your `README` file, right now your issue page link results in the `404` error page, which means that the link you provided for the `issue` page of the repository is not correct. Kindly fix the issue and make sure that your links are working properly. Here with "links" I mean not only this `issue` link but others too.

![image](https://github.com/hydaline-code/expense_tracker/assets/66476329/274cfcd4-c26b-4c27-b217-4fa7654634dd)